packages/cubejs-server-core/core/CompilerApi.js

```diff
@@ -58,7 +58,10 @@ class CompilerApi {
     options = options || {};
     const { includeDebugInfo } = options;
     const dbType = this.getDbType();
+
+    console.log('DB TYPE:', dbType);
```

Comment:
Can you please remove this debug line? Thanks

Reply:
Whoops! Removed them